.tekton/ansible-ai-connect-service-push.yaml

```diff
@@ -271,7 +271,7 @@ spec:
       - name: TARGET_STAGE
         value: production
       - name: BUILD_ARGS
-        value: ["IMAGE_TAGS=latest 1.0.$(tasks.git-metadata.results.commit-timestamp)", "GIT_COMMIT=$(tasks.clone-repository.results.commit)"]
```

Comment:
when will this tag be applied? we need this tag to be applied when snyk check passes so that the image autoupdates will pick this up.

Reply:
please see the `apply-tags` task and the condition added into it.

Reply:
@jameswnl in `apply-tags` section: 
```yaml
 - name: apply-tags
      [...]
      - name: ADDITIONAL_TAGS
        value: ["pr-{{pull_request_number}}", "pr-{{pull_request_number}}.$(tasks.git-metadata.results.commit-timestamp)"]
      [...]
      when:
        - input: $(tasks.sast-snyk-check.results.TEST_OUTPUT)
          operator: in
          values:
            - '"result":"SUCCESS"'